website/docs/docs/build/custom-aliases.md

```diff
@@ -6,18 +6,22 @@ id: "custom-aliases"
 
 ## Overview
 
-When dbt runs a model, it will generally create a relation (either a `table` or a `view`) in the database. By default, dbt uses the filename of the model as the identifier for this relation in the database. This identifier can optionally be overridden using the [`alias`](/reference/resource-configs/alias) model configuration.
+When dbt runs a model, it will generally create a relation (either a <Term id="table" /> or a <Term id="view" />) in the database, or in the case of an [ephemeral model](/docs/build/materializations#ephemeral) it will create a <Term id="cte" /> for use in another model. By default, dbt uses the filename of the model as the identifier for the relation or CTE it creates. This identifier can optionally be overridden using the [`alias`](/reference/resource-configs/alias) model configuration.
 
 ### Why alias model names?
 The names of schemas and tables are effectively the "user interface" of your <Term id="data-warehouse" />. Well-named schemas and tables can help provide clarity and direction for consumers of this data. In combination with [custom schemas](/docs/build/custom-schemas), model aliasing is a powerful mechanism for designing your warehouse.
 
-### Usage
-The `alias` config can be used to change the name of a model's identifier in the database. The following <Term id="table" /> shows examples of database identifiers for models both with, and without, a supplied `alias`.
```

Comment:
I tweaked this line to remove the misleading `<Term>` component, since in this case we're not actually talking about a relation (which is what the term documents), we're just referencing the markup table that follows.

Reply:
Good call! Thank you